app.py

```diff
@@ -87,6 +93,7 @@ def nearby_qa(_location):
 
 
 @app.route("/register", methods=["GET", "POST"])
+@auth_secret_key
```

Comment:
We can drop authentication on this endpoint. Its okay for anonymous CM to be able to register libraries, in order for any CM that are registered to be visible in the app they have to be approved in the library registry admin interface.

I think we can drop the whole `secret_key` auth.